2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -103,6 +119,7 @@ collector {
     }
 
     # Incoming events are stored in a buffer before being sent to Kinesis/Kafka.
+    # Note: Buffering is not supported by NSQ; will be ignored    
```

Comment:
what do you mean?

- http://nsq.io/clients/tcp_protocol_spec.html#identify
- https://github.com/brainlag/JavaNSQClient/blob/master/src/main/java/com/github/brainlag/nsq/NSQCommand.java#L44-L54
- https://github.com/brainlag/JavaNSQClient/blob/9f5d767eb5e3495ebc032eae711fba7929fed132/src/main/java/com/github/brainlag/nsq/Connection.java#L72